src/linter/ui5Types/TypeLinter.ts

```diff
@@ -183,6 +196,21 @@ export default class TypeLinter {
 		}
 		typeCheckDone();
 
+		// Provide TS Compiler information to the XMLViews i.e. controllers, etc.
+		const rawLintResults = this.#context.generateRawLintResults();
+		for (const {filePath, rawMessages} of rawLintResults) {
+			// Process only XML files
+			if (!filePath.endsWith("xml")) {
+				continue;
+			}
+
+			rawMessages.forEach((message) => {
+				if (message.fix && message.fix instanceof EventHandlersFix) {
+					message.fix.methodExistsInController(this.#context, program, checker);
+				}
+			});
+		}
+
 		this.#sharedLanguageService.release();
 
 		this.addMessagesToContext();
```

Comment:
Isn't it possible to add the iteration over the fixes into `addMessagesToContext`, which already iterates over all messages?